src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const correctState = { ...state };
+  let clone = {};
+  const result = [];
+
+  for (const action of actions) {
+    clone = {};
```

Comment:
i think u can add object `let clone` one time, in 15 line

Reply:
thx